src/errorpage.cc

```diff
@@ -180,20 +180,30 @@ error_hard_text[] = {
         ERR_REQUEST_START_TIMEOUT,
         "request start timedout"
     },
+    {
+        ERR_REQUEST_PARSE_TIMEOUT,
+        "request parse timedout"
+    },
+    {
+        ERR_RELAY_REMOTE,
+        "relay server response"
+    }
+}};
```

Comment:
AFAICT, adding ERR_REQUEST_PARSE_TIMEOUT/ERR_RELAY_REMOTE to this container and the conversion of this container to std::array are the only improvements worth keeping in this PR after master/v6 commit 29304c7.  We should keep errorFindHardText() name/interface (while updating its implementation) IMO.

@eduard-bagdasaryan, please update this PR branch and PR title/description accordingly, removing the current Draft mark from the PR.

Reply:
Done.